chromium_src/net/base/url_util.cc

```diff
@@ -80,4 +82,16 @@ bool ParseAuthHostAndPort(base::StringPiece input,
   return true;  // Success.
 }
 
+std::string URLToEphemeralStorageDomain(const GURL& url) {
```

Comment:
I don't think this method is needed here https://github.com/brave/brave-core/pull/7154/files#r524857285

Reply:
Where do you suggest goes in a way that it can be used in the `CookieMonster` and also in `EphemeralTabStorageHelper` or do you prefer to have two version of this code?

Reply:
I don't think we need two versions of the code if we just pass in the url and only convert it to ephemeral storage domain when we actually create it. The code would only ever be called from one file

Reply:
This is also called in `EphemeralStorageTabHelper` to manage DOM storage. We need to make sure the conversion from URL to storage domain is in sync.